### PR TITLE
Throws a meaningful warning if there is a mismatch between sql and options for generate inclusion statistics

### PR DESCRIPTION
**Pull request by gowthamrao**
Throws a meaningful warning if there is a mismatch between sql and options for generate inclusion statistics

Fixed https://github.com/OHDSI/CohortDiagnostics/issues/35

### CODE REVIEW

File: R/CohortConstruction.R [hunk 1]
```diff
@@ -627,4 +629,25 @@ saveAndDropTempInclusionStatsTables <- function(connection,
...
+}
+
+
+.warnMismatchSqlInclusionStats <- function(sql) {
+  if (any(stringr::str_detect(string = sql, pattern = "_inclusion_result"), 
+          stringr::str_detect(string = sql, pattern = "_inclusion_stats"), 
+          stringr::str_detect(string = sql, pattern = "_summary_stats")
+          )
+  ) {
+    if (isFALSE(generateInclusionStats)) {
```
Comment (schuemie) [inline R/CohortConstruction.R:641]: Are you referring to a global variable (`generateInclusionStats`) here?!?

This should be an argument of the `.warnMismatchSqlInclusionStats` function

**Comment by schuemie** [comment]
You did not run R check on this. I can tell, because you're failing the [automated R check on travis](https://travis-ci.com/github/OHDSI/CohortDiagnostics/builds/160902505). Please always run R check before making a pull request.

In this case the fail is because you're using stringr, but didn't declare that dependency in the DESCRIPTION file.

File: R/CohortConstruction.R (hunk 1 above)
Comment (gowthamrao) [reply R/CohortConstruction.R:641]: Thank you !

Comment (gowthamrao) [reply R/CohortConstruction.R:641]: Thank you! 

```
.warnMismatchSqlInclusionStats: no visible binding for global variable
  â€˜generateInclusionStatsâ€™
launchCohortExplorer: no visible binding for global variable
  â€˜shinySettingsâ€™
Undefined global functions or variables:
  generateInclusionStats shinySettings
```

```
* checking dependencies in R code ... WARNING
'::' or ':::' import not declared from: â€˜stringrâ€™
```

**Pull request merged**